trpcultivate_genetics/config/install/trpcultivate_genetics.settings.yml

```diff
@@ -0,0 +1,4 @@
+terms:
+  sample_germplasm_relationship_type: 0
+  sample_type: 5
```

Comment:
This looks like a default set for testing? You should leave this as 0 and then change in your test. Otherwise you are assigning a random default at this point for the full module ;-p